external/jenkins/README.md

```diff
@@ -0,0 +1,31 @@
+## External (Third Party Container) Tests
+
+Third Party container tests help verify that the AdoptOpenJDK binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
```

Comment:
Please replace the beginning of the sentence with:
Jenkins tests are part of the external third-party application tests that help verify...
instead of 
Third Party container tests help verify...

Reply:
Add a sentence at the end that says (with the hyperlink to the repo):
Jenkins test material is pulled from the [Jenkins CI repository](https://github.com/jenkinsci/jenkins.git)